docs/api/session.md

```diff
@@ -356,7 +356,7 @@ Returns `String` - The user agent for this session.
 * `callback` Function
   * `result` Buffer - Blob data.
 
-Returns `Blob` - The blob data associated with the `identifier`.
```

Comment:
Looking at these docs, doesn't this method return nothing 🤔  The callback has the actual data not the return value

Reply:
That seemed weird to me.  I should have digged deeper:

https://github.com/electron/electron/blob/95cb601f2b903ab137417d09afa86bfb6e103dd1/atom/browser/api/atom_api_session.cc#L640

Reply:
Tested the above change and the generated d.ts looks good 
`getBlobData(identifier: string, callback: (result: Buffer) => void): void;`

Reply:
@MarshallOfSound can you recheck?  Thanks